src/core/client-app.ts

```diff
@@ -311,13 +327,15 @@ export class ClientApp
     private _configureCoreServices(): void
     {
         this._container
-            .registerInstance("DialogService", new DefaultDialogService(this._accentColor))
             .registerInstance("EventAggregator", new DefaultEventAggregator())
             .registerInstance("NavigationService", new DefaultNavigationService(this._pageManager.vueRouterInstance))
             .registerInstance("StorageService", new DefaultStorageService())
             .registerInstance("DisplayService", new DefaultDisplayService())
             .registerInstance("ComponentService", new DefaultComponentService())
             ;
+        
+        if (!this._isDialogServiceRegistered)
+            this._container.registerInstance("DialogService", new DefaultDialogService({ accentColor: this._accentColor }));
```

Comment:
Move the this._accentColor as the fallback value inside the DefaultDialogService itself